lib/src/crawl_async.dart

```diff
@@ -26,25 +26,25 @@ final _empty = Future<Null>.value(null);
 /// the graph. If missing nodes are important they should be tracked within the
 /// [readNode] callback.
 ///
-/// If either [readNode] or [children] throws the error will be forwarded
+/// If either [readNode] or [edges] throws the error will be forwarded
 /// through the result stream and no further nodes will be crawled, though some
 /// work may have already been started.
 Stream<V> crawlAsync<K, V>(Iterable<K> roots, FutureOr<V> Function(K) readNode,
-    FutureOr<Iterable<K>> Function(K, V) children) {
-  final crawl = _CrawlAsync(roots, readNode, children)..run();
+    FutureOr<Iterable<K>> Function(K, V) edges) {
```

Comment:
I think this could maybe also change to be 1 generic param with the 2 new args?

If you want to submit as is I can take a crack at that before we publish.

Reply:
ack